build.gradle

```diff
@@ -74,30 +75,31 @@ repositories {
 }
 
 dependencies {
-    compile 'org.springframework.boot:spring-boot-starter-data-jpa'
-    compile 'org.springframework.boot:spring-boot-starter-web'
-    compile 'org.springframework.boot:spring-boot-starter-security'
-    compile 'org.springframework.boot:spring-boot-configuration-processor'
+    compile 'org.springframework.boot:spring-boot-starter-data-jpa:2.2.6.RELEASE'
```

Comment:
Je hoeft de versienummers hier niet te herhalen, daar zorgt de dependency management plug-in voor (zie Spring Boot docs welke dependencies er allemaal worden meegenomen)

Reply:
Ik heb de versienummers weggehaald.